tests/test_oc4ids_tranforms.py

```diff
@@ -0,0 +1,1332 @@
+    assert output["contractingProcesses"][4]["summary"]["status"] == "active"
+
+
+def test_contract_status_closed():
+
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "tender": {"status": "cancelled"},
+        },
+        {
+            "ocid": "ocds-213czf-2",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "awards": [{"status": "cancelled"}],
+            "tender": {"id": 1},
+        },
+        {
+            "ocid": "ocds-213czf-3",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "contracts": [{"status": "cancelled"}],
+            "tender": {"id": 1},
+        },
+        {
+            "ocid": "ocds-213czf-4",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "contracts": [{"period": {"endDate": "2000-01-01"}}],
+            "awards": [{"contractPeriod": {"endDate": "2000-01-01"}}],
+            "tender": {"id": 1, "contractPeriod": {"endDate": "2000-01-01"}},
+        },
+        {
+            "ocid": "ocds-213czf-5",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "contracts": [{"period": {"endDate": "3000-01-01"}}],
+            "awards": [{"contractPeriod": {"endDate": "2000-01-01"}}],
+            "tender": {"id": 1, "contractPeriod": {"endDate": "2000-01-01"}},
+        },
+        {
+            "ocid": "ocds-213czf-6",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "contracts": [{"period": {"endDate": "2000-01-01"}}],
+            "awards": [{"contractPeriod": {"endDate": "3000-01-01"}}],
+            "tender": {"id": 1, "contractPeriod": {"endDate": "2000-01-01"}},
+        },
+        {
+            "ocid": "ocds-213czf-7",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "contracts": [{"period": {"endDate": "2000-01-01"}}],
+            "awards": [{"contractPeriod": {"endDate": "2000-01-01"}}],
+            "tender": {"id": 1, "contractPeriod": {"endDate": "3000-01-01"}},
+        },
+    ]
+
+    output = transforms._run_transforms(
+        copy.deepcopy(releases), "1", transforms=[transforms.contracting_process_setup, transforms.contract_status],
+    )
+
+    assert output["contractingProcesses"][0]["summary"]["status"] == "closed"
+    assert output["contractingProcesses"][1]["summary"]["status"] == "closed"
+    assert output["contractingProcesses"][2]["summary"]["status"] == "closed"
+    assert output["contractingProcesses"][3]["summary"]["status"] == "closed"
+    assert output["contractingProcesses"][4]["summary"]["status"] != "closed"
```

Comment:
Should this be 'active'? (At least one contract period (whether expressed as Contract.period, Award.contractPeriod or tender.contractPeriod) includes the present date.)

Reply:
Also for the next two assertions